### PR TITLE
chore(ci): remove integrations from docs smoke tests

### PR DESCRIPTION
**Pull request by lilnasy**
## Changes
- Should be unnecessary after #9479

## Testing
Does not affect behavior

## Docs
Does not affect usage

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -234,17 +234,3 @@ jobs:
...
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SOURCE_REPO: ${{ github.event.pull_request.head.repo.full_name || github.event.repository.full_name }}
-          SOURCE_BRANCH: ${{ github.head_ref || github.ref_name }}
-
-      - name: Test
-        run: pnpm run test:smoke
-        env:
-          SKIP_OG: true
-          PUBLIC_TWO_LANG: true
```
Comment (TheOtterlord) [inline .github/workflows/ci.yml:250]: We don't need to remove the entire smoke test. Just the `docgen:integrations` generation. I can't inline suggest, but here's what it should probably look like now

```
      - name: Build autogenerated docs pages from current astro branch
        if: ${{ steps.changes.outputs.docs == 'true' }}
        run: cd smoke/docs && pnpm docgen && pnpm docgen:errors
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          SOURCE_REPO: ${{ github.event.pull_request.head.repo.full_name || github.event.repository.full_name }}
          SOURCE_BRANCH: ${{ github.head_ref || github.ref_name }}

      - name: Test
        run: pnpm run test:smoke
        env:
          SKIP_OG: true
          PUBLIC_TWO_LANG: true
```

File: package.json [hunk 2]
```diff
@@ -21,9 +21,7 @@
     "test:match": "cd packages/astro && pnpm run test:match",
     "test:unit": "cd packages/astro && pnpm run test:unit",
     "test:unit:match": "cd packages/astro && pnpm run test:unit:match",
-    "test:smoke": "pnpm test:smoke:example && pnpm test:smoke:docs",
-    "test:smoke:example": "turbo run build --concurrency=100% --filter=\"@example/*\"",
-    "test:smoke:docs": "turbo run build --filter=docs",
```
Comment (TheOtterlord) [inline package.json:26]: Same as my other comment. The smoke test is fine, it's just one step of the test.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lilnasy) [reply .github/workflows/ci.yml]: 🥳 passing again

**Review by TheOtterlord — APPROVED** [review]
LGTM! Thanks

**Pull request merged**